taker/src/main.rs

```diff
@@ -126,10 +126,10 @@ struct Opts {
     #[clap(subcommand)]
     network: Option<Network>,
 
-    #[clap(short, long, parse(try_from_str = parse_umbrel_seed))]
-    umbrel_seed: Option<[u8; 32]>,
+    #[clap(short, long, parse(try_from_str = parse_app_seed))]
+    app_seed: Option<[u8; 32]>,
```

Comment:
Is the term `app` redundant?

Reply:
not really, we have more than one seed used in our app (and this is the name used in some of these distros)

Reply:
If this is true, then `app` is not precise enough. What other seeds do we have?

Reply:
wallet seed?

Reply:
That's the same thing... given you are in the `taker` binary it might be enough to call `seed`, because `taker` + `seed` = `taker_seed`, which is the same naming we use in the filesystem :) 

Eitherway, no harm done. 🙃